mautic/mautic
# Form custom attributes

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | x
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Some software may require custom attributes for form tag (analytics etc).
This PR added custom attributes for form tag. Similar like we've already use custom attributes for fields.

![image](https://user-images.githubusercontent.com/462477/61200374-76a6b500-a6e1-11e9-9e59-586c61af3e4f.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form and add Form attributes (new field )
3. Check the source code of form and you should see custom attributes in form tag

![image](https://user-images.githubusercontent.com/462477/61200446-b1105200-a6e1-11e9-91f0-f86063642faf.png)
